cli/dts/lib.deno.unstable.d.ts

```diff
@@ -480,7 +482,10 @@ declare namespace Deno {
    * The result of `Deno.emit()` API.
    */
   export interface EmitResult {
-    /** Diagnostic messages returned from the type checker (`tsc`). */
+    /** Diagnostic messages returned from the type checker (`tsc`).
+     * 
```

Comment:
@ebebbington trailing whitespace here :(

Reply:
thanks, will get around to it soon 👍 busy with work n stuff

Reply:
https://github.com/denoland/deno/pull/10925/commits/22ceb73cb26983544e9ac979159a52d437425c14